lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -164,6 +167,22 @@ class LighthouseViewerReport {
       return id;
     }).catch(err => logger.log(err.message));
   }
+
+  /**
+   * Enables pasting a JSON report on the page.
+   */
+  onPaste(e) {
+    e.preventDefault();
+
+    ga('send', 'event', 'report', 'paste');
+
+    try {
+      const json = JSON.parse(e.clipboardData.getData('text'));
+      this.replaceReportHTML(json);
+    } catch (err) {
+      // noop
```

Comment:
Not sure it will be that useful to users. I'd prefer just ignoring random key pressing or users that accidentally paste something they don't mean to.  Note: `replaceReportHTML` will show a butter bar if the json file is invalid.